### PR TITLE
[Component with its own class name as service type] LPS-201312 Refactor Resource components from Commerce module

### PR DESCRIPTION
**Pull request by MrJohn1911**
Previous pull: https://github.com/liferay-core-infra/liferay-portal/pull/6134

Ticket: https://liferay.atlassian.net/browse/LPS-201312

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:commerce

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

File: modules/apps/commerce/commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/util/BaseCommerceResource.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+		int start = (page - 1) * pageSize;
+		int end = page * pageSize;
+
+		List<AccountEntry> userAccountEntries =
+			accountEntryLocalService.getUserAccountEntries(
+				userId, parentAccountId, keywords,
+				commerceAccountHelper.toAccountEntryTypes(commerceSiteType),
+				commerceAccountHelper.toAccountEntryStatus(true), start, end);
+
+		for (AccountEntry accountEntry : userAccountEntries) {
```
Comment (dantewang) [inline modules/apps/commerce/commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/util/BaseCommerceResource.java:73]: @MrJohn1911 Please check whether you can simplify with `TransformUtil`.

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:commerce

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:commerce

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
The last change in this pull was just an update in the bnd.bnd file

**Pull request closed without merging**